### ISSUE TITLE
`$cfg['ShowGitRevision']` no longer works

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#18627]
### Describe the bug

`$cfg['ShowGitRevision']` no longer works, since last commit 960f7d90a8558000704b5799de3745ac8fe1a83a. Neither on demo servers or localhost.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/ or
https://demo.phpmyadmin.net/master/public/index.php?route=/
2. See missing git info

### Expected behavior

`$cfg['ShowGitRevision'] = true;` should show the git commit info.

### Server configuration

- phpMyAdmin version: 5.2.2-dev, 6.0.0-dev

**Comment by SachinBahukhandi** [comment]
Works on the latest stable release I suppose:
![image](https://github.com/phpmyadmin/phpmyadmin/assets/17425177/885c3e00-4111-4deb-acdd-b1590cdd042c)

**Comment by liviuconcioiu** [comment]
Now it works on 6.0.0-dev, earlier it didn't, but still there is no link to the commit 5b5e444. I've noticed a few days ago that there's no longer a link to the commit.

**Comment by liviuconcioiu** [comment]
Now on 6.0.0-dev, isn't showing the git information in the footer.

![git](https://github.com/phpmyadmin/phpmyadmin/assets/25424343/5f697976-dde7-4748-b666-8aba1c0f6555)

**Comment by williamdes** [comment]
> https://www.phpmyadmin.net/api/commit/5b5e444adfb36905d3f04b19d973d5d26e60407d/ returns 502 Bad Gateway.
> 
> @williamdes can you look into this? Probably is related to the latest changes on the website?

Interesting, Thanks for the ping I will have a look

**Comment by liviuconcioiu** [comment]
> Now on 6.0.0-dev, isn't showing the git information in the footer.

I think this one is related to #18667. Is missing since then.

**Comment by williamdes** [comment]
I am forwarding the 502 issue to https://github.com/phpmyadmin/website/issues/163

**Comment by williamdes** [comment]
Fixed, finally !!
Does it work better now ?

**Comment by liviuconcioiu** [comment]
> Fixed, finally !! Does it work better now ?

Yes. This issue is fixed. Not related to this one, on 6.0.0-dev, in the footer the revision is no longer showing, and I think it's related to https://github.com/phpmyadmin/phpmyadmin/pull/18667.

**Comment by williamdes** [comment]
Did it get fixed ?

**Comment by liviuconcioiu** [comment]
> Did it get fixed ?

The initial issue is fixed, but there's another one in 6.0.0 (in the footer the git revision is not showing). The problem is that I've tested the code on localhost and works perfectly, but on demo server it isn't.

![issue](https://github.com/phpmyadmin/phpmyadmin/assets/25424343/7f2e766f-0203-49fb-91bb-0c144d031b8f)

**Comment by liviuconcioiu** [comment]
@williamdes You can close this one, as I've tested with `bin/console write-revision-info` and the information is written on localhost. Probably there's something wrong on the demo server. I'll open another issue with a few other things that are related to the demo server only, at https://github.com/phpmyadmin/website

**Comment by williamdes** [comment]
I can move this issue for /scripts repo. Good for you ?

**Comment by liviuconcioiu** [comment]
> I can move this issue for /scripts repo. Good for you ?

Is best if I open another one, since the initial issue has been fixed and the other one is not related.

**Comment by liviuconcioiu** [comment]
Closing this one. Git info seems to be fixed. Probably by the latest commit https://github.com/phpmyadmin/scripts/commit/affad722b566bffc62bb1dd2fb25f27ad6668944
